README.md

```diff
@@ -1,69 +1,40 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
 
-> Hello Microverse! project.
+> In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. My goal here is to master all of the tools and best practices i have learned about in previous steps. I will be using them in all Microverse projects and most likely in my future job as well.
 
 ## Built With
 
 - Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- HTML
+- CSS
+- npm
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
+- VS Code
+- chrome
 
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
+## Setup
+git clone https://github.com/developerbaalla/Hello-Microverse.git
 
+cd Hello-Microverse
 
+code.
 
 ## Authors
 
 👤 **Author1**
```

Comment:
- [ ] You should use the author name in place of the placeholder text **Author1**.